src/raftstore/store/store.rs

```diff
@@ -1290,6 +1290,63 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         // we will call the callback with timeout error.
     }
 
+    fn propose_batch_raft_snapshot_command(&mut self,
+                                           batch: Vec<RaftCmdRequest>,
+                                           on_finished: BatchCallback) {
+        let size = batch.len();
+        let mut ret: Vec<Option<RaftCmdResponse>> = vec![None; size];
+        let mut batch_sizes: HashMap<_, u64> = HashMap::default();
```

Comment:
We can remove this, `self.raft_metrics.propose.batch_local_read.observe(batch.len() as u64 as f64);` is enough.